Classes/Neos/Neos/Ui/Domain/Model/Changes/AbstractCreate.php

```diff
@@ -176,6 +213,20 @@ protected function createNode(NodeInterface $parent)
             $node->setProperty($key, $value);
         }
 
+        if ($nodeType->isOfType('TYPO3.Neos:Content') && $this->getReferenceData()['subject']['fusionPath']) {
```

Comment:
Is there actually any scenario, in which we create content nodes outside of content collections?